### DISCUSSION TITLE
FormItem的错误反馈布局的一些想法

### DISCUSSION THREAD

**Discussion by henryzp** [alibaba/formily#2717] [Q&A 2.x]
保存的时候，会这样的气泡提示：

![image](https://user-images.githubusercontent.com/3849702/147810557-5d436c35-2327-4686-b67c-a6cb00fdca61.png)

其实比较好的交互是不是应该保存的时候，只显示红框，没有气泡。。当鼠标移到红框时，错误的信息以气泡层的形式显示出来？

**Comment by janryWang** [comment]
应该可以，欢迎来个PR

**Reply by henryzp** [reply]
我先想想

**Comment by henryzp** [comment]
@janryWang 

![image](https://user-images.githubusercontent.com/3849702/148639268-d0c4951f-33da-4fae-9ee3-2bb187dab0dc.png)

其实只要注释掉这个visible，就行了，但我又不能保证其他的小伙伴是不是能接受这样的交互。。

亦或者是再提供一个什么配置？

**Reply by janryWang** [reply]
肯定要提供配置的，不能Break Change

**Reply by henryzp** [reply]
feedbackShow:
- always
- mouseover
- click

这个api可以么，默认是always，或者你有没有更好的？
